tinygrad/nn/optim.py

```diff
@@ -76,9 +58,12 @@ def step(self) -> None:
       m_hat = self.m[i] / (1.0 - self.b1**self.t)
```

Comment:
Is this why better than the way the Adam does it?

Reply:
If I do this the way the Adam implementation does it, it breaks for LAMB

Reply:
Meh, this way I find more readable anyway, and it's a wash in the JIT.